generics/helpers/utils.js

```diff
@@ -213,6 +214,31 @@ function convertStringToBoolean(stringData) {
   return stringToBoolean;
 }
 
+/**
+  * md5 hash
+  * @function
+  * @name md5Hash
+  * @returns {String} returns uuid.  
+*/
+
+function md5Hash(value) {
+  return md5(value);
+}
+
```

Comment:
@priyanka-TL  Js doc is missing for this function.

Reply:
@aman-tunerlabs js doc is there 
/**
  * md5 hash
  * @function
  * @name md5Hash
  * @returns {String} returns uuid.  
*/

Reply:
@aman-tunerlabs js doc is there /**
  * md5 hash
  * @function
  * @name md5Hash
  * @returns {String} returns uuid.  
*/

Reply:
@priyanka-TL  JS doc not present for generateLink function